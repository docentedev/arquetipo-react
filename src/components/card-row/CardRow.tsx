import React from 'react';

import './CardRow.css';

/**
 * @author Claudio Rojas <claudio.dcv@gmail.com>
 * @since v0.1.0
 * @version v1.0.0
 * @param props
 */
const CardRow : React.FC<React.Props<any>> = (props) => (
  <div className="card-row">{props.children}</div>
);
export default CardRow;