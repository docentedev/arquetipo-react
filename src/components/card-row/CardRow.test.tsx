import React from 'react';
import CardRow from './CardRow';
import renderer from 'react-test-renderer';

test('CardRow is ok renderer', () => {
    const component = renderer.create(
        <CardRow />,
      );
      let tree = component.toJSON();
      expect(tree).toMatchSnapshot();
});