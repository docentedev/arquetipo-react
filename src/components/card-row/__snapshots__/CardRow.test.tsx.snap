// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardRow is ok renderer 1`] = `
<div
  className="card-row"
/>
`;
